packages/header-checker-lint/__snapshots__/header-checker-lint.js

```diff
@@ -1,19 +1,3 @@
-/**
```

Comment:
Looks like snap-shot-it strips this out.

Reply:
Yeah, I'd noticed this, any thoughts? is there a heuristic way we could skip `__snapshot__` files, while still catching files that should have headers; it's been helpful to me catching a few legacy files that were simply missing headers.

Reply:
I'm looking into the configuration files in #34